src/main/java/codeu/model/data/Tag.java

```diff
@@ -0,0 +1,39 @@
+package codeu.model.data;
+
+import java.time.Instant;
+import java.util.UUID;
+
+/** Class representing a tag. */
+public class Tag {
+  private final UUID id;
+  private final String tag;
+  private final Instant creation;
+
+  /**
+   * Constructs a new Tag.
+   *
+   * @param id the ID of this Tag
+   * @param tag the content of this Tag
+   * @param creation the creation time of this Tag
+   */
+  public Tag(UUID id, String tag, Instant creation) {
+    this.id = id;
+    this.tag = tag;
```

Comment:
What is the ID used for, as opposed to what the String tag is used for? I feel like it's a bit redundant.

Reply:
I think the ID would be the UUID for each tag, while the actual string tag would be like the input that the user provides. it might be redundant? but maybe helpful in identifying them apart from their string contents